content/competitions/hackathon-2021.html

```diff
@@ -0,0 +1,121 @@
+---
+title: "Hackathon"
+draft: false
+---
+
+<div class="section columns is-centered">
+    <section class="column is-6">
+        
+        <div class="block">
+            <a href="https://www.facebook.com/events/200547242027862/">
+                <img src="{{< baseurl >}}/img/media/competitions/hackathon_2021_banner.png" style="width: 100%" />
```

Comment:
If and only if #257 is merged, this line will need to change to:
```suggestion
                <img src="{{< relURL "/img/media/competitions/hackathon_2021_banner.png" >}}" style="width: 100%" />
```
or simply
```suggestion
                <img src="/img/media/competitions/hackathon_2021_banner.png" style="width: 100%" />
```
The current site uses a mix of both and in our environment, they will render the same. The top line is the "direct" translation from baseurl and is what I changed Hackathon 2020 to, but the bottom line will work fine too and matches other relative links on this 2021 page.

If your PR is merged first, then I will amend my PR to make this change so nothing breaks.